src/reducers/images.js

```diff
@@ -12,17 +12,23 @@ import {
   ADD_FRAME,
   UPDATE_GIF_PROGRESS,
   ADD_GIF,
+  UPDATE_GIF_FILENAME,
   UPDATE_IMAGE_SETTING,
   UPDATE_BOUNDS_SETTING,
   UPDATE_STRATEGY,
-  RESET
+  RESET,
+  UPDATE_TEXT,
+  UPDATE_TEXT_COLOR
 } from '../constants/action-types';
 
 const initialState = {
   frames: {},
   frameIDs: [],
   gifProgress: 0,
-  gifData: ''
+  gifData: '',
+  caption: '',
+  fontColor: '#0000',
```

Comment:
I assume this was a typo for `#000`. Looks like the color picker always generates six-digit hex colors, so let's use `#000000` here as well.